lisa/runners/lisa_runner.py

```diff
@@ -776,17 +776,21 @@ def _merge_test_requirements(
                             and hasattr(self, "platform")
                             and self.platform.runbook.ignored_capability
                         ):
-                            node_requirement.features.items = [
-                                x
-                                for x in node_requirement.features
-                                if str(x).lower()
-                                not in list(
+                            ignored_features = []
+                            for feature in node_requirement.features:
+                                if str(feature).lower() in list(
                                     map(
                                         str.lower,
                                         self.platform.runbook.ignored_capability,
                                     )
+                                ):
+                                    node_requirement.features.items.remove(feature)
+                                    ignored_features.append(feature)
+                            if ignored_features:
+                                self._log.debug(
```

Comment:
Make sure it's logged only once in the whole log, because it's used to know if a feature string is effective or not. It can be like below, but you can keep to improve the string.

"the ignored feature 'aaa' is used.."

Reply:
the ignore feature is based on test case requirement, it will be different results for different test cases, once for the whole log is insufficient.

I feel "the ignored feature 'aaa' is used.." makes users confused